tdrs-backend/tdpservice/users/test/test_auth.py

```diff
@@ -247,7 +249,7 @@ def test_login_with_general_exception(mocker):
 def test_login_with_inactive_user(mocker, api_client, inactive_user):
     """Login with inactive user should error and return message."""
     os.environ["JWT_KEY"] = test_private_key
-    inactive_user.username = "test@example.com"
+    inactive_user.username = "test_inactive@example.com"
```

Comment:
These and others below were renamed to fix an IntegrityError; the `sub` values were updated to account for new auth method.